lib/WasmReader/WasmBytecodeGenerator.cpp

```diff
@@ -322,12 +325,12 @@ WasmBytecodeGenerator::EmitExpr(WasmOp op)
         uint8 depth = m_reader->m_currentNode.br.depth;
         if (depth >= m_labels->Count())
             Assert(UNREACHED);
+        SListCounted<Js::ByteCodeLabel>::Iterator itr(m_labels);
+        itr.Next();
         for (int i = 0; i < depth; i++) {
-            // [b-gekua] TODO
-            // Find label at depth nesting levels out
-            // Possibly the SList m_labels is not optimal for this.
+            itr.Next();
```

Comment:
Server Error in '/' Application.

A potentially dangerous Request.Form value was detected from the client (ctl00$ctl00$ctl00$ctl00$cphRoot$cphSite$cphLC$cphC$txtCandidate_Notes_Web="...Maghsoudi <dralireza.maghsoudi...").
Description: Request Validation has detected a potentially dangerous client input value, and processing of the request has been aborted. This value may indicate an attempt to compromise the security of your application, such as a cross-site scripting attack. You can disable request validation by setting validateRequest=false in the Page directive or in the configuration section. However, it is strongly recommended that your application explicitly check all inputs in this case. 

Exception Details: System.Web.HttpRequestValidationException: A potentially dangerous Request.Form value was detected from the client (ctl00$ctl00$ctl00$ctl00$cphRoot$cphSite$cphLC$cphC$txtCandidate_Notes_Web="...Maghsoudi <dralireza.maghsoudi...").

Source Error: 

[No relevant source lines]

Source File: c:\Windows\Microsoft.NET\Framework64\v2.0.50727\Temporary ASP.NET Files\root\a24ecd5a\8d0ac35\App_Web_registerdetails.aspx.2587f01f.5ikejyg9.0.cs    Line: 0 

Stack Trace: 

[HttpRequestValidationException (0x80004005): A potentially dangerous Request.Form value was detected from the client (ctl00$ctl00$ctl00$ctl00$cphRoot$cphSite$cphLC$cphC$txtCandidate_Notes_Web="...Maghsoudi <dralireza.maghsoudi...").]
   System.Web.HttpRequest.ValidateString(String s, String valueName, String collectionName) +11188987
   System.Web.HttpRequest.ValidateNameValueCollection(NameValueCollection nvc, String collectionName) +71
   System.Web.HttpRequest.get_Form() +178
   System.Web.HttpRequest.get_HasForm() +11189223
   System.Web.UI.Page.GetCollectionBasedOnMethod(Boolean dontReturnNull) +124
   System.Web.UI.Page.DeterminePostBackMode() +83
   System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint) +11155303
   System.Web.UI.Page.ProcessRequest(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint) +11154842
   System.Web.UI.Page.ProcessRequest() +91
   System.Web.UI.Page.ProcessRequest(HttpContext context) +240
   ASP.candidate_register_registerdetails_aspx.ProcessRequest(HttpContext context) in c:\Windows\Microsoft.NET\Framework64\v2.0.50727\Temporary ASP.NET Files\root\a24ecd5a\8d0ac35\App_Web_registerdetails.aspx.2587f01f.5ikejyg9.0.cs:0
   System.Web.CallHandlerExecutionStep.System.Web.HttpApplication.IExecutionStep.Execute() +599
   System.Web.HttpApplication.ExecuteStep(IExecutionStep step, Boolean& completedSynchronously) +171


Version Information: Microsoft .NET Framework Version:2.0.50727.5485; ASP.NET Version:2.0.50727.5491